.travis.yml

```diff
@@ -22,7 +23,8 @@ env:
 
 matrix:
   include:
-    - env: TOXENV=translations SQLALCHEMY_DATABASE_URI="$SQLALCHEMY_DATABASE_TEST_URI" PERSISTENCE_FILE="https://raw.githubusercontent.com/uwcirg/TrueNTH-USA-site-config/develop/site_persistence_file.json"
+    - env: TOXENV=translations PERSISTENCE_FILE="https://raw.githubusercontent.com/uwcirg/TrueNTH-USA-site-config/develop/site_persistence_file.json"
+    - env: TOXENV=translations PERSISTENCE_FILE="https://raw.githubusercontent.com/uwcirg/ePROMs-site-config/develop/site_persistence_file.json"
```

Comment:
I don't think PERSISTENCE_FILE is set by default anywhere so those two rows should be the only time it's being set in TravisCI. Each row should run as a separate TravisCI "job" with those env vars set